usb_ser_mon/usb_ser_mon.py

```diff
@@ -358,7 +359,7 @@ def main():
 
         # Otherwise wait for the teensy device to connect
         while True:
-            dev = {}
+            dev = namedtuple('Device', ['properties'])({})
```

Comment:
This change is out of place with the rest of the PR.

I don't see any advantage to using a namedtuple here.

Reply:
Just two lines below we are trying to use `dev.properties`:

`    dev.properties['ID_SERIAL_SHORT'] = args.serial
`

that would fail if `dev` is a dictionary.

Reply:
Also, the namedtuple change is already in the master branch. This PR affects only the line 383.

Reply:
Ahh - ok - it looked like that commit included with this PR.